sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -718,6 +718,10 @@ ALTER TABLE procedure_order ADD COLUMN history_order enum('0','1') DEFAULT '0';
        ALTER TABLE `amc_misc_data` add column `soc_provided` DATETIME default NULL;
 #EndIf
 
+#IfMissingColumn form_encouter pos_code
+       ALTER TABLE `form_encounter` ADD `pos_code` INT(2) NOT NULL ;
```

Comment:
Have you tested this?  It's not clear to me how the ALTER TABLE can work with existing rows if there is no default value and nulls are disallowed.